Link E2E example to docs (#1790)
zenml-io/zenml: docs/book/user-guide/starter-guide/using-project-templates.md

# example: zenml init --template e2e_batch --template-with-defaults
```

## Project templates used in our examples

<!-- ### Starter Guide

Our Starter Guide documentation is built around the `Starter` project template codes. 

> Mostly examples will come from it, so we highly recommend you to install `starter` template

Reviewer: ```suggestion
Most examples will be based on it, so we highly recommend you to install the `starter` template
```

with `--template-with-defaults` flag before diving deeper into this documentation section, 
so you can follow this guide along using your own local environment.

```bash
mkdir starter
cd starter
zenml init --template starter --template-with-defaults
``` -->

### Advanced Guide

> Our Advanced Guide documentation is built around the `E2E Batch` project template codes. 
> Mostly examples will come from it, so we highly recommend you to install `e2e_batch` template
> with `--template-with-defaults` flag before diving deeper into this documentation section, 
> so you can follow this guide along using your own local environment.

Reviewer: ```suggestion
Our Advanced Guide documentation is built around the `E2E Batch` project template codes. 
Most examples will be based on it, so we highly recommend you to install the `e2e_batch` template
with `--template-with-defaults` flag before diving deeper into this documentation section, 
so you can follow this guide along using your own local environment.
```
